conf.py

```diff
@@ -16,7 +16,7 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('.'))
+#sys.path.insert(0, os.path.abspath('.'))
```

Comment:
If we don't like this, we can add a guard that checks the [`READTHEDOCS` environment variable]( http://docs.readthedocs.io/en/latest/faq.html#how-do-i-change-behavior-for-read-the-docs ).